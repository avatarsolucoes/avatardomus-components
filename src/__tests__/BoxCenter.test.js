import React from 'react';
// import renderer from 'react-test-renderer';
import { shallow } from '../setupEnzyme';
import { BoxCenter } from '..';

describe('BoxCenter', () => {
  it('is truthy', () => {
    expect(BoxCenter).toBeTruthy();
  });

  it('Test renderer without children', () => {
    const wrapper = shallow(<BoxCenter className="test" />);
    expect(wrapper.exists('.test')).toBe(true);
    expect(wrapper.html()).toMatchSnapshot();
  });

  it('Test renderer whith children', () => {
    const wrapper = shallow(<BoxCenter>TESTE</BoxCenter>).html();
    expect(wrapper).toMatchSnapshot();
  });
});
