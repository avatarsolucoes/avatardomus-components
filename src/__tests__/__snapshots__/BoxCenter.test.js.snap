// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BoxCenter Test renderer whith children 1`] = `"<div class=\\"boxCenterColumn\\"><div class=\\"boxCenterLine\\">TESTE</div></div>"`;

exports[`BoxCenter Test renderer without children 1`] = `"<div class=\\"boxCenterColumn test\\"><div class=\\"boxCenterLine\\"></div></div>"`;
